src/test/java/uk/gov/companieshouse/missingimagedelivery/orders/api/controller/GlobalExceptionHandlerTest.java

```diff
@@ -6,9 +6,8 @@
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.*;
```

Comment:
we want to avoid using wildcard imports :)

Reply:
Sorted these out